usermods/Tubes/node.h

```diff
@@ -98,50 +122,66 @@ class LightNode {
 
     MessageReceiver *receiver;
     MeshNodeHeader header;
+
+    typedef enum{
+        NODE_STATUS_QUIET=0,
+        NODE_STATUS_STARTING,
+        NODE_STATUS_RECEIVING,
+        NODE_STATUS_STARTED,
+        NODE_STATUS_MAX,
+    } NodeStatus;
     NodeStatus status = NODE_STATUS_QUIET;
 
-    char node_name[20];
+    const char* status_code() {
+        switch (status) {
+        case NODE_STATUS_QUIET:
+            return " (quiet)";
+        case NODE_STATUS_STARTING:
+            return " (starting)";
+        case NODE_STATUS_RECEIVING:
```

Comment:
this adds an additional state during startup when a node is receiving, but waiting to broadcast based on the statusTimer

Reply:
renamed to  be more clear of network capabilities